nix/pkgs/firecracker-rootfs/extra-files/etc/systemd/system/debug-shell.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
```

Comment:
is not not copyrighted anymore?

Reply:
I wrote this. I copied firecracker-agent.service to debug-shell.service. But there is firecracker agent left.